README.md

```diff
@@ -10,9 +10,10 @@ Goals:
 
 ## Installation notes
 
-This assumes you are on MacOS with HomeBrew installed. `make install` will brew install `gitleaks`. The install will:
+`make install` will install `gitleaks`. The install will:
```

Comment:
It wasn't accurate that we were installing `gitleaks` via brew. We were downloading it directly from a release artifact on Github.